src/shared/ui/BurgerMenuIcon/index.tsx

```diff
@@ -0,0 +1,2 @@
+import BurgerMenuIcon from "./BurgerMenuIcon";
+export default BurgerMenuIcon;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.